src/stores/EpgController.ts

```diff
@@ -152,6 +94,17 @@ export default class EpgService {
     } as EpgChannel;
   };
 
+  getEpgService = (item: PlaylistItem) => {
+    const scheduleType = item?.scheduleType || EPG_TYPE.JWP;
+    const service = getNamedModule(EpgService, scheduleType);
+
+    if (!service) {
+      throw Error(`No epg service was added for the ${scheduleType} schedule type`);
```

Comment:
Will this cause the whole app to crash if someone mis-types the epg type? If so, can we make it more resiliant and just log a warning adn return an empty schedule?

Reply:
Makes sense to me